src/common/camera_control.h

```diff
@@ -22,6 +22,12 @@
 #include "common/darktable.h"
 
 #include <glib.h>
+
+#if defined (_WIN32)
```

Comment:
A general remark, you are using `_WIN32`, elsewhere we already have `__WIN32__` and `WIN32` and `G_OS_WIN32`. Would it be possible to find out in how far all those differ and change the code to only use one, provided that's possible and there aren't subtle differences when each of those are defined and relevant in the places they are used?